apps/explorer/src/app/routes/oracles/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { gql, useQuery } from '@apollo/client';
+import type { OracleSpecs as OracleSpecsQuery } from './__generated__/OracleSpecs';
+
+import React from 'react';
+import { SyntaxHighlighter } from '@vegaprotocol/ui-toolkit';
+import { RouteTitle } from '../../components/route-title';
+import { t } from '@vegaprotocol/react-helpers';
+import { SubHeading } from '../../components/sub-heading';
+
+const ORACLE_SPECS_QUERY = gql`
+  query OracleSpecs {
+    oracleSpecs {
+      status
+      id
+      createdAt
+      updatedAt
+      pubKeys
+      filters {
+        key {
+          name
+          type
+        }
+        conditions {
+          value
+          operator
+        }
+      }
+      data {
+        pubKeys
+      }
+    }
+  }
+`;
+
+const Oracles = () => {
+  const { data } = useQuery<OracleSpecsQuery>(ORACLE_SPECS_QUERY);
```

Comment:
question (non blocking): I notice we aren't error handling here or for other queries in explorer?

Reply:
Yep, something we need to address!